javascript/todo/src/controls.js

```diff
@@ -0,0 +1,21 @@
+export default class Controls {
+  #titles;
```

Comment:
No need for private properties, they aren't inherited so we won't have direct access for title and description properties